ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -247,11 +286,11 @@ public void sendRequest(
         method,
         RequestBodyUtil.createProgressRequest(
           multipartBuilder.build(),
-          new ProgressRequestListener() {
+          new ProgressListener() {
         long last = System.nanoTime();
 
         @Override
-        public void onRequestProgress(long bytesWritten, long contentLength, boolean done) {
+        public void onProgress(long bytesWritten, long contentLength, boolean done) {
```

Comment:
Yes, `ProgressRequestListener.onRequestProgress` was previously only used for upload progress. I am generalizing that interface to be useful for any kind of progress, both request *and* response. And yes, request upload progress still works.